client/reader/controller.js

```diff
@@ -238,43 +238,6 @@ export function blogListing( context, next ) {
 	next();
 }
 
-export function userListing( context, next ) {
-	const userId = context.params.user_id;
-
-	if ( userId.trim().length === 0 ) {
-		next();
-		return;
-	}
-
-	const basePath = '/read/users/:user_id';
-	const fullAnalyticsPageTitle = analyticsPageTitle + ' > User > ' + userId;
-	const mcKey = 'blog';
-
-	trackPageLoad( basePath, fullAnalyticsPageTitle, mcKey );
-	recordTrack( 'calypso_reader_user_profile', { user_id: userId } );
-
-	context.primary = (
-		<AsyncLoad
-			require="calypso/reader/user-stream"
-			key={ 'user-' + userId }
-			streamKey={ 'user:' + userId }
-			userId={ userId }
-			trackScrollPage={ trackScrollPage.bind(
-				null,
-				basePath,
-				fullAnalyticsPageTitle,
-				analyticsPageTitle,
-				mcKey
-			) }
-			onUpdatesShown={ trackUpdatesLoaded.bind( null, mcKey ) }
-			suppressSiteNameLink
-			showBack={ userHasHistory( context ) }
-			placeholder={ null }
-		/>
-	);
-	next();
-}
-
```

Comment:
Since we'll need a handful of these, I moved this to a new file in /user-stream/controller.tsx.